<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>721</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane { /* The tab widget frame */
    border-top: 2px solid #C2C7CB;
    
	background-color: rgb(255, 255, 255);
}

QTabWidget::tab-bar {
    left: 5px; /* move to the right by 5px */
    
	color: rgb(0, 0, 127);
    }

/* Style the tab using the tab sub-control. Note that
    it reads QTabBar not QTabWidget */
QTabBar::tab {
 
	background-color: rgb(23, 54, 93);
    
	color: rgb(255, 255, 255);
    border: 2px solid #C4C4C3;
    border-bottom-color: #C2C7CB; /* same as the pane color */
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 8ex;
    padding: 2px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
	
   
}

QTabBar::tab:selected {
    border-color: #9B9B9B;
    border-bottom-color: #C2C7CB; /* same as pane color */
}

QTabBar::tab:!selected {
    margin-top: 2px; /* make non-selected tabs look smaller */
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Tab">
     <attribute name="title">
      <string>Ajouter logement</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>701</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox  {
    border: 1px solid gray;
    border-color: #FF17365D;
    margin-top: 27px;
    font-size: 14px;
    border-radius: 15px;
	background-color: rgb(255, 255, 255);
}

QGroupBox::title  {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 5px 8000px 5px 8000px;
    background-color: #FF17365D;
    color: rgb(255, 255, 255);
}
QLabel {
	font: 11pt &quot;Adobe Gothic Std B&quot;;
}
QLabel{
	color: rgb(23, 54, 93);
}
QLineEdit
{
 
    background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: gray;
    font:  10px;
    min-width: 10 em;
    padding: 6px;

 }
QComboBox
{
    subcontrol-origin: padding;
    subcontrol-position: top right;
    selection-background-color: #111;
    selection-color: yellow;
    color: white;
    
	background-color: rgb(100, 175, 194);
    border-style: solid;
    border: 2px solid ;
    
	border-color: rgb(192, 192, 192);
    border-radius: 5;
    padding: 1px 0px 1px 20px;
}


</string>
      </property>
      <property name="title">
       <string>Logement</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: rgb(23, 54, 93);
}
</string>
       </property>
       <property name="text">
        <string>ref_log</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>49</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>129</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>1000000010</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>100</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: rgb(23, 54, 93);
}
</string>
       </property>
       <property name="text">
        <string>type_log</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>140</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>adresse_log</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>36</x>
         <y>180</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>surface_log</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>260</y>
         <width>142</width>
         <height>31</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    color: rgb(255, 255, 255);
  background-color: #FF17365D;
    border-style: outset;
    border-width: 0px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
}
QPushButton:hover {
color: #fff;
    background-color: rgb(43, 163, 255);
    border-style: inset;
}
QPushButton:pressed {
    background-color: rgb(43, 163, 255);
    border-style: inset;
}</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>100</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>duplex</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>villa</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>appartement</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="lineEdit_17">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>209</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_18">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>249</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_30">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>220</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>piece_log</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_31">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>260</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>dispo_log</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_16">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>169</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>446</x>
         <y>53</y>
         <width>191</width>
         <height>181</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../Downloads/LOGO RE.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>-24</x>
        <y>-7</y>
        <width>751</width>
        <height>401</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../Downloads/photo-1528460033278-a6ba57020470.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>label_13</zorder>
     <zorder>groupBox_2</zorder>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Modifier logement</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>691</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox  {
    border: 1px solid gray;
    border-color: #FF17365D;
    margin-top: 27px;
    font-size: 14px;
    border-radius: 15px;
	background-color: rgb(255, 255, 255);
}

QGroupBox::title  {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 5px 8000px 5px 8000px;
    background-color: #FF17365D;
    color: rgb(255, 255, 255);
}
QLabel {
	font: 11pt &quot;Adobe Gothic Std B&quot;;
}
QLabel{
	color: rgb(23, 54, 93);
}
QLineEdit
{
 
    background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: gray;
    font:  10px;
    min-width: 10 em;
    padding: 6px;

 }
QComboBox
{
    subcontrol-origin: padding;
    subcontrol-position: top right;
    selection-background-color: #111;
    selection-color: yellow;
    color: white;
    
	background-color: rgb(100, 175, 194);
    border-style: solid;
    border: 2px solid ;
    
	border-color: rgb(192, 192, 192);
    border-radius: 5;
    padding: 1px 0px 1px 20px;
}


</string>
      </property>
      <property name="title">
       <string>Logement</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="text">
        <string>ref_log</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>49</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>129</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>100</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>type_log</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>140</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>adresse_log</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>36</x>
         <y>180</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>surface_log</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>270</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    color: rgb(255, 255, 255);
  background-color: #FF17365D;
    border-style: outset;
    border-width: 0px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
}
QPushButton:hover {
color: #fff;
    background-color: rgb(43, 163, 255);
    border-style: inset;
}
QPushButton:pressed {
    background-color: rgb(43, 163, 255);
    border-style: inset;
}</string>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>100</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>duplex</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>villa</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>appartement</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="lineEdit_19">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>169</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_32">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>230</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>piece_log</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_33">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>270</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>dispo_log</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_20">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>219</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_21">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>259</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>60</y>
         <width>191</width>
         <height>181</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../Downloads/LOGO RE.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>-50</x>
        <y>-270</y>
        <width>771</width>
        <height>641</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../Downloads/photo-1528460033278-a6ba57020470.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>label_14</zorder>
     <zorder>groupBox_3</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Supprimer logement</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>701</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox  {
    border: 1px solid gray;
    border-color: #FF17365D;
    margin-top: 27px;
    font-size: 14px;
    border-radius: 15px;
	background-color: rgb(255, 255, 255);
}

QGroupBox::title  {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 5px 8000px 5px 8000px;
    background-color: #FF17365D;
    color: rgb(255, 255, 255);
}
QLabel {
	font: 11pt &quot;Adobe Gothic Std B&quot;;
}
QLabel{
	color: rgb(23, 54, 93);
}
QLineEdit
{
 
    background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: gray;
    font:  10px;
    min-width: 10 em;
    padding: 6px;

 }
QComboBox
{
    subcontrol-origin: padding;
    subcontrol-position: top right;
    selection-background-color: #111;
    selection-color: yellow;
    color: white;
    
	background-color: rgb(100, 175, 194);
    border-style: solid;
    border: 2px solid ;
    
	border-color: rgb(192, 192, 192);
    border-radius: 5;
    padding: 1px 0px 1px 20px;
}


</string>
      </property>
      <property name="title">
       <string>Logement</string>
      </property>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="text">
        <string>ref_log</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>49</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>260</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    color: rgb(255, 255, 255);
  background-color: #FF17365D;
    border-style: outset;
    border-width: 0px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
}
QPushButton:hover {
color: #fff;
    background-color: rgb(43, 163, 255);
    border-style: inset;
}
QPushButton:pressed {
    background-color: rgb(43, 163, 255);
    border-style: inset;
}</string>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>50</y>
         <width>191</width>
         <height>181</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../Downloads/LOGO RE.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-1</y>
        <width>721</width>
        <height>391</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../Downloads/photo-1528460033278-a6ba57020470.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>label_15</zorder>
     <zorder>groupBox_4</zorder>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Afficher logement</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>21</y>
        <width>671</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
